app/controllers/api/v2/registration_commands_controller.rb

```diff
@@ -21,6 +21,7 @@ class RegistrationCommandsController < V2::BaseController
         param :update_packages, :bool, desc: N_("Update all packages on the host")
         param :repo, String, desc: N_("Repository URL / details, for example for Debian OS family: 'deb http://deb.example.com/ buster 1.0', for Red Hat and SUSE OS family: 'http://yum.theforeman.org/client/latest/el8/x86_64/'")
         param :repo_gpg_key_url, String, desc: N_("URL of the GPG key for the repository")
+        param :use_wget, :bool, desc: N_("Use wget instead of curl")
```

Comment:
Rather than a boolean parameter, would it make sense to use an enum that defaults to `curl` but also accepts `wget` just to be explicit?

Reply:
I agree. I will change the boolean to an enum and replace the checkbox in the Host Registration page by a `FormSelect` field defaulting to `curl`.

Reply:
Done :)